static/css/style.css

```diff
@@ -563,6 +567,10 @@ blockquote
     left:0;
   }
 */
+  .vimeo-video
+  {
+    display: none;
```

Comment:
It just looked funny to me when I made the window smaller. Maybe you can show me what a better solution would look like.